tests/src/python/test_provider_ogr_gpkg.py

```diff
@@ -999,8 +999,8 @@ def testGeopackageManyLayers(self):
         count = count_opened_filedescriptors(tmpfile)
         if count > 0:
             # We should have just 1 but for obscure reasons
-            # uniqueFields() leaves one behind
-            self.assertEqual(count, 2)
+            # uniqueFields() (sometimes?) leaves one behind
+            self.assertTrue(count == 1 or count == 2)
```

Comment:
```suggestion
            self.assertIn(count, (1, 2))
```

Reply:
(will give more helpful output if the test fails)

Reply:
(I miss pytest. you can write assertions in a natural way)